MODULE.md

```diff
@@ -0,0 +1,262 @@
+  - `maxEntries`: The maximum number of entries.
+  - `historicalEntries`: The number of historical entries to store.
+  - `bondDenom`: The denomination for bonding and staking.
+  - `minCommissionRate`: The minimum commission rate for validators.
+
+#### Administrative Rights:
+
+- Assign or revoke administrative privileges.
+- Determine if validators have the ability to self-revoke.
+
+#### Commands
+
+##### Update Parameters (update-params):
+
+- Syntax: `manifestd tx poa update-params [admin1,admin2,admin3,...] [allow-validator-self-exit-bool]`
+
+  - Parameters:
+    - `admin1,admin2,admin3,...`: A list of admin addresses which can be multisig addresses.
+    - `allow-validator-self-exit-bool`: A boolean value (true or false) to allow validators to self-exit.
+
+  **Example:** `manifestd tx poa update-params manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct,manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct false`
+
+##### Update Staking Parameters (update-staking-params):
+
+- Syntax: `manifestd tx poa update-staking-params [unbondingTime] [maxVals] [maxEntries] [historicalEntries] [bondDenom] [minCommissionRate]`
+
+  - Parameters:
+
+    - `unbondingTime`: The amount of time it takes to unbond tokens.
+    - `maxVals`: The maximum number of validators.
+    - `maxEntries`: The maximum number of entries.
+    - `historicalEntries`: The number of historical entries to store.
+    - `bondDenom`: The denomination for bonding and staking.
+    - `minCommissionRate`: The minimum commission rate for validators.
+
+  **Example:** `manifestd tx poa update-staking-params 1814400 100 7 1000 umfx 0.01`
+
+##### Set Voting Power (set-power):
+
+- Syntax: `manifestd tx poa set-power [validator] [power] [--unsafe]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+    - `power`: The voting power to give the validator.
+    - `--unsafe`: A flag to allow the PoA admin to set the voting power of a validator that is not in the active set.
+
+    **Example:** `manifestd tx poa set-power manifestvaloper1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct 1000000000 --unsafe`
+
+##### Remove Pending Validator (remove-pending):
+
+- Syntax: `manifestd tx poa remove-pending [validator]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+
+    **Example:** `manifestd tx poa remove-pending manifestvaloper1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct`
+
+##### Remove Validator (remove):
+
+- Syntax: `manifestd tx poa remove [validator]`
+
+  - Parameters:
+
+    - `validator`: The validator's operator address.
+
+    **Example:** `manifestd tx poa remove manifestvaloper1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct`
+
+## Token Factory Module
+
+The Token Factory module as it is implemented on the Manifest Network, allows the PoA admin to have granular control over the creation and management of tokens on the Manifest Network. The admin can mint, burn, edit, and transfer tokens to other accounts from any account.
```

Comment:
Does that mean that the admin can send tokens from a user account to another user account without user approval?

In this context, does admin == PoA admin?

Reply:
In all contexts, the PoA admin is the admin of a tokenfactory token because only the PoA admin can create them

Reply:
yes, this is apart of the Cosmos tokenfactory spec. I can disable this forced-transfer logic if you like in 1 line: https://github.com/liftedinit/manifest-ledger/blob/main/app/app.go#L163

As mentioned in another comment, the tokenfactory admin is the creator of the token, and can be changed by only the admin. Anyone can create new tokens.